modules/extension/xsd/xsd-wps/src/main/java/org/geotools/wps/v2_0/bindings/DataInputTypeBinding.java

```diff
@@ -0,0 +1,102 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2021, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.wps.v2_0.bindings;
+
+import javax.xml.namespace.QName;
+import net.opengis.wps20.DataInputType;
+import net.opengis.wps20.Wps20Factory;
+import org.geotools.wps.v2_0.WPS;
+import org.geotools.xsd.AbstractComplexEMFBinding;
+
+/**
+ * Binding object for the type http://www.opengis.net/wps/2.0:DataInputType.
+ *
+ * <p>
+ *
+ * <pre>
+ *  <code>
+ *  &lt;?xml version="1.0" encoding="UTF-8"?&gt;&lt;complexType name="DataInputType" xmlns="http://www.w3.org/2001/XMLSchema"&gt;
+ *
+ *  		&lt;annotation&gt;
+ *
+ *  			&lt;documentation&gt;
+ *
+ *  				This structure contains information elements to supply input data for process execution.
+ *
+ *  			&lt;/documentation&gt;
+ *
+ *  		&lt;/annotation&gt;
+ *
+ *  		&lt;sequence&gt;
+ *
+ *  			&lt;choice&gt;
+ *
+ *  				&lt;element ref="wps:Data"/&gt;
+ *
+ *  				&lt;element ref="wps:Reference"/&gt;
+ *
+ *  				&lt;element maxOccurs="unbounded" name="Input" type="wps:DataInputType"/&gt;
+ *
+ *  			&lt;/choice&gt;
+ *
+ *  		&lt;/sequence&gt;
+ *
+ *  		&lt;attribute name="id" type="anyURI" use="required"&gt;
+ *
+ *  			&lt;annotation&gt;
+ *
+ *  				&lt;documentation&gt;
+ *
+ *  					Identifier of this input.
+ *
+ *  				&lt;/documentation&gt;
+ *
+ *  			&lt;/annotation&gt;
+ *
+ *  		&lt;/attribute&gt;
+ *
+ *  	&lt;/complexType&gt;
+ *
+ *   </code>
+ * </pre>
+ *
+ * @generated
+ */
+public class DataInputTypeBinding extends AbstractComplexEMFBinding {
```

Comment:
Most of these classes are adding nothing but noise, they could be replaced by an instance of ComplexEMFBinding.
See the WPS 1.0 configuraton and bindings, only a few bindings are implemneted as classes, all of the others are just instantiated in the configuration.

Reply:
Class removed. Use org.geotools.xsd.ComplexEMFBinding in WPSConfiguration to reduce number of bindings class